### ISSUE TITLE
Git remote code execution

### ISSUE THREAD

**Issue by vbjay** [gitextensions/gitextensions#4999]
https://www.bleepingcomputer.com/news/security/malicious-git-repository-can-lead-to-code-execution-on-remote-systems/

Fyi and a suggestion to highly suggest users are using at least 2.17.1 version of git.

**Comment by sharwell** [comment]
This was resolved in #4933

**Comment by veremenko-y** [comment]
@sharwell  I have 2.51.01 and when I check for updates:

![image](https://user-images.githubusercontent.com/4127660/40729065-ca22c6d4-63e8-11e8-879c-b5f461c91c49.png)

**Comment by vbjay** [comment]
Not exactly. I'm saying we should suggest they update if they don't have at
least 2.17.1. This is more related to the automatic update and git version
checks.

**Comment by vbjay** [comment]
@veremenko-y This isn't about GE updates.  This is about GIT version checks and the fact that earlier versions have a remote code execution CVE that was fixed in git 2.17.1.  I created this issue to help broadcast that fact and to suggest our git version checks strongly suggest they update because of that fact.

**Comment by vbjay** [comment]
See #4932

**Comment by veremenko-y** [comment]
@vbjay I am sorry, it is my fault. I missed the point, that git was removed from Git Extensions installation package. I downloaded the GE installer after I wrote the message.

**Comment by vbjay** [comment]
@veremenko-y  https://git-scm.com/download/

**Comment by veremenko-y** [comment]
@vbjay Thank you, I already did this. My concern with auto update was with presumption that Git is still bundled with Git Extensions, but it is not anymore.

**Comment by vbjay** [comment]
Correct.  We have a feature issue on the way to handle git updates in the
installer and GE to handle these kinds of situations but it is not merged
yet.  #4967.
